addons/mail/static/src/discuss/core/common/channel_invitation.js

```diff
@@ -35,6 +36,7 @@ export class ChannelInvitation extends Component {
             searchResultCount: 0,
             searchStr: "",
         });
+        this.fetchPartnersToInvite = useDebounced(this.fetchPartnersToInvite.bind(this), 250);
```

Comment:
I don't think we want to delay the one in `onWillStart` that load initial values (quickly as there is no filtering).

Reply:
Indeed we just want to debounce the `onInput`